CMSIS/RTOS/RTX/SRC/rt_CMSIS.c

```diff
@@ -269,7 +269,7 @@ static inline  t __##f (t1 a1, t2 a2, t3 a3, t4 a4) {                          \
 
 #elif defined (__ICCARM__)      /* IAR Compiler */
 
-#define __NO_RETURN __noreturn
+#define __NO_RETURN __attribute__((__noreturn__))
```

Comment:
Is this change backward compatible with all versions of IAR compiler? Or would this break anything?
RTX4 is deprecated. Why do you think its worth maintaining it? Any customer demands?